### DISCUSSION TITLE
Support for constraints (`~>`, `>=`, etc)?

### DISCUSSION THREAD

**Discussion by rktjmp** [lumen-oss/rocks.nvim#403] [Q&A]
Does rocks.nvim support semver constraints? I assumed it would be delegated to luarocks, which does [appear to support constraints (under dependencies)](https://github.com/luarocks/luarocks/wiki/Rockspec-format#dependency-information) but `norg = "~> 7"` (and variations) fails with a "version number '~> 7' could not be parsed" error  during a `Rocks sync`.

Is it a planned feature?

**Comment by mrcjkb** [comment]
I hope I can plan it soon 😄 

It would be blocked by https://github.com/luarocks/luarocks/issues/1684, which I might start working on myself.

I've created an issue: https://github.com/nvim-neorocks/rocks.nvim/issues/404
